api_backend.yaml

```diff
@@ -1010,6 +1032,26 @@ definitions:
           - name
           - family_name
       - $ref: "#/definitions/LimitedFederatedUser"
+  TokenType:
+    type: string
+    description: The type of the provided access token.
+    x-extensible-enum:
+      - Bearer
+  SupportToken:
+    title: Support token
+    description: A Support Token response
+    type: object
+    properties:
+      access_token:
+        type: string
+      expires_in:
+        type: number
+      token_type:
+        $ref: "#/definitions/TokenType"
```

Comment:
can we have different token types?

Reply:
Nope, for this moment :)

Reply:
why we need to return this value then?

Reply:
we base our implementation on this standard:
```
{
    "access_token": "{your access token}",
    "expires_in": {15 minutes expressed as seconds},
    "token_type": "Bearer"
}
```

Reply:
why?

Reply:
The proposed interface refers to the Auth0 solution. We only use Bearer token, for simplicity we can omit this parameter.

Reply:
Yes, as a general guidelines a parameter must be _motivated_ and _documented_ otherwise we are going to accumulate tech debt on things that we don't remember the raison d'etre of.